static/app/api/urlpatterns.generated.ts

```diff
@@ -0,0 +1,794 @@
+/**
+ * GENERATED FILE. Do not edit manually.
+ * To update it run `python3 -m tools.api_urls_to_typescript`
+ */
+
+type KnownApiUrls = [
+  '/',
+  '/',
```

Comment:
should this be a giant union instead of an array?

type 
```suggestion
type KnownApiUrls = 
  '/' | '/issues/' | '/accept-transfer/';
```

Reply:
we have the array mainly to be able to add response types or query param types to it at one point. I hope extracting the union on TS level is still “cheap”. If not, we can go to directly generating a union - [on 97115, I did that](https://github.com/getsentry/sentry/pull/97155/files#diff-4ef1c2315253e3d4f501d8925952a34a661f948f416b40570b2f5beda9dbc562R5-R6).